### ISSUE TITLE
First aid and BHV personal details

### ISSUE THREAD

**Issue by BrendaH** [evolution-events/Artaxerxes#68]
Do we want to know if participants have a First Aid or BHV certificate (and are willing to use these skills at our event)?
If so, where do we ask for this info? With personal details seems logical, but the certificate should still be valid for the event they're attending (we could ask for 'valid until' info, which needs updating for every preparation of registration).
However, it might be that for some event they would want to serve as first aider or BHV-er but for other events not. If we want to make that a possibility we'd have to put it with a registration (perhaps as one of the options). Then perhaps the requirement 'need to stay sober' needs to be mentioned.

It is probably most convenient to only ask for First Aid since we have quite some BHV-ers in our organisation (and it is not mandatory to have a BHV-er on hand) and at the event discuss with any first aiders if they want to stay sober (possibly in shifts).
Question remains: where and how do we ask this info?

**Comment by ReneB** [comment]
We could - and maybe should - ask if they _have_ it but personally, I'm not on board with allowing for a "I could, but I won't" option, because if someone needs help and I know someone with a certificate for that, I really don't feel like taking "no" for an answer. They can self-select by not telling us about the certificate, of course, but any more than that is not realistic, I think.

"Willing to stay sober" is, of course, a different ballgame. We shouldn't demand people to remain sober just because they have a certificate, and we could ask them but if no one's willing to remain sober, where does that leave us? Do we *need* someone to remain sober (aside from a driver, of course!)

I personally feel it's a property of a person, but there is something to be said for auto-refreshing the data by making it part of the registration.

**Comment by matthijskooijman** [comment]
> We could - and maybe should - ask if they have it but personally, I'm not on board with allowing for a "I could, but I won't" option, because if someone needs help and I know someone with a certificate for that, I really don't feel like taking "no" for an answer. They can self-select by not telling us about the certificate, of course, but any more than that is not realistic, I think.

It all depends on what the implications are of asking someone to act as a first-aider on the event. When something happens and a first-aider is nearby, they'll likely help out anyway. However, it's a different thing to pull someone out of their game on the far side of the terrain to help someone, or to ask someone to remain sober, etc. I think it is acceptable for someone to not want to be on standby as a first-aider on an event and instead let the organisers arrange someone else for that (as might be considered their responsibility).

> "Willing to stay sober" is, of course, a different ballgame. We shouldn't demand people to remain sober just because they have a certificate, and we could ask them but if no one's willing to remain sober, where does that leave us? Do we need someone to remain sober (aside from a driver, of course!)

If we are serious about having an "on-duty" first-aider, then we might also require them to be sober and we might need to ask an external first-aider in case none are available (which is essentially the same as "none want to remain sober", if we think that is important). I'm not sure if we should care much about soberness, though, nor if we as organizers should be responsible for having first-aid present (depends a bit on the size of the event, of course). Regardless, even if do not consider ourselves responsible, it might be convenient to know in advance whether there will be first-aiders.

> I personally feel it's a property of a person, but there is something to be said for auto-refreshing the data by making it part of the registration.

If we care about validity of certificates (which I think we don't, unless we really run into regulatory requirements on BHV or similar, which I think only applies to bigger events), then that would also be easier as part of the registration (since you can then ask for a valid certificate at the date of the event).


In any case, I think this is something that is out of scope for the MVP: We're currently doing things without this information, so we can certainly continue doing so in the near future :-)

**Comment by ReneB** [comment]
I definitely agree that this is out of scope for MVP, but just to put in another $0.02: I think it's good to distinguish between "we know this person can do first aid" - a matter of information gathering - and "we expect this person to attend as a first aider" - a matter of crew management. So yes, I agree that it is most definitely acceptable to not want to be "the BHV'er" for an event - but if there is no first aider present, or we don't know where they are, we will ask you for help anyway if we feel it's _required_ (that latter part is important: we won't be asking you to apply a band-aid, of course).

**Comment by matthijskooijman** [comment]
Good point: That would mean that "Are you a first-aider" (or something similar) is a good general question, and then we could (as part of the event preparation) contact these people to see to what degree they are willing to help. Possibly the UI should offer some info about this process (with the risk of us forgetting to contact people and making assumptions, but well :-p)
